### ISSUE TITLE
not presenting on new chrome canary

### ISSUE THREAD

**Issue by feiss** [aframevr/aframe#3272]
I've just downloaded Chrome Canary, enabled WebVR flags, and tried A-Frame samples, but although pose seems to work, it doesn't present on the headset, it just goes fullscreen. I tried webvr.info and threejs examples (https://threejs.org/examples/?q=vr#webvr_cubes) and they work.

- A-Frame Version: aframe.io samples
- Platform / Device: win64, htc vive
- Reproducible Code Snippet or URL: http://aframe.io  samples

**Comment by dmarcos** [comment]
I can reproduce the same problem.

**Comment by machenmusik** [comment]
(Even when the OpenVR flag is thrown?)

**Comment by machenmusik** [comment]
I wonder if this is A-Frame taking too long to set up like the vr2vr traversal issues earlier?

**Comment by vt5491** [comment]
I have also been experiencing this problem over the last several months.  I just assumed it was a chrome canary issue since firefox worked fine.  However, I was testing with a native three.js app and realized that chrome canary worked fine.   Then I searched google and found this ticket.

Anyway as of 2018-02-20, I did a git pull, and using 'dist/aframe-master.js', I am now able to enter VR with aframe and chrome canary (not the 'aframe-v0.7.1.js though).   I have problems with either the enter-vr button not showing or not working, so I have to press the "f" key to enter VR.  I also don't seem to get split-screen console tracking in the browser.  But when I put on the HMD, I see everything with tracking.  I have an oculus, but I see the characteristic "steam/vive vr circle", which indicates to me it's  probably doing Vive emulation of some sort.  Also, the default orientation is off (I have to look to the side to see the image) but aside from these issues everything seems to work.
